scripts/13-ocp-iam-users.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
I think you can make the script more generic and not IBM specific.

Like adding new users to use the toolkit, just need to provide a list of user IDs.
I think you can still make a prefix to an userid optional and for IBM they can pass a prefix "IAM#"

Reply:
maybe name script `xx-add-new-toolkit-users.sh`